blogging/admin.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib import admin
+from blogging.models import Post, Category
+
+# Register your models here.
+# admin.site.register(Post)
+# admin.site.register(Category)
+#
+class CategoryInLine(admin.StackedInline):
+    model = Category.posts.through
+
+class PostAdmin(admin.ModelAdmin):
+    inlines = [CategoryInLine]
+    model = Post
```

Comment:
I think this may be redundant or implied when you run 
admin.site.register(Post, PostAdmin)

but I like that it makes the model used obvious!